styles/styles.css

```diff
@@ -0,0 +1,451 @@
+    flex-basis: 400px;
+    width: 100%;
+}
+.artical-2-foto{
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 400px;
+    width: 100%;
+
+}
+.div-1{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 300px;
+    height: 200px;
+    margin: 60px 40px;
+}
+.div-1 h3{
+    font-size: 2rem;
+}
+@media (max-width: 650px) {
+    .div-1{
+        gap: 20px;
+    }
+}
+.line{
+    width:80px;
+}
+@media (max-width: 650px) {
+    .line{
+        width:50px;
+    }
+}
+
+img {
+    width: 100%;
+    height: 100%;
+}
+span{
+    flex-basis: 800px;
+    flex-wrap: wrap;
+    box-shadow: 1px 4px 20px rgb(75, 74, 74);
+
+
+}
+@media (max-width: 900px) {
+    .hero-2 {
+        flex-direction: column;
+        gap: 2rem;
+    }
+
+    .artical-2-foto {
+        flex-direction: column;
+        justify-content: flex-end;
+        gap: 1rem;
+    }
+    .artical-foto {
+        flex-direction: column-reverse;
+        justify-content: flex-end;
+        gap: 1rem;
+    }
+
+    .div-1 {
+        width: 300px;
+        height: 200px;
+        font-size: 1rem;
+    }
+
+    span {
+        flex-basis: auto;
+    }
+}
+
+@media (max-width: 400px) {
+    .hero-2 {
+        width: 300px;
+    }
+
+    .div-1 {
+        font-size: 1rem;
+        width: 50%;
+    }
+}
+
+@media (max-width: 480px) {
+
+    .div-1 {
+        width: 90%;
+        margin: 20px 10px;
+    }
+
+    .artical-foto, .artical-2-foto {
+        gap: 0.5rem;
+    }
+
+    img {
+        height: 100%;
+    }
+}
+
+.container-1{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-content: center;
+
+}
+
+.hero-3{
+    display: grid;
+    grid-template-columns: repeat(3, 300px);
+    grid-template-rows: repeat(2, 300px);
+    grid-gap: 30px;
+    justify-content: center;
+    align-content: center;
+    padding: 40px 40px;
+
+}
+
+
+.artical-web{
+    border:#5722CD 2px solid;
+    color: rgba(75, 74, 74, 0.8);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    font-size: 13px;
+    padding: 20px;
+    box-shadow: 1px 5px 20px rgba(182, 179, 179, 0.56);
+}
+h4{
+    font-size: 1.5rem;
+    color: #5722CD;
+}
+.logo{
+    height: 70px;
+    width: 70px;
+}
+
+@media (max-width: 1024px) {
+    .hero-3 {
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: auto;
+    }
+}
+
+@media (max-width: 768px) {
+    .hero-3 {
+        grid-template-columns: 1fr;
+        grid-template-rows: auto;
+    }
+}
+
+@media (max-width: 480px) {
+    .artical-web {
+        font-size: 12px;
+        padding: 10px;
+    }
+
+    .logo {
+        height: 50px;
+        width: 50px;
+    }
+}
+
+.container-2{
+    background-image: linear-gradient(to right, #4B49E8, #5722CD);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 800px;
+}
+
+form{
```

Comment:
Je gebruikt hier een type-selector. Dat is prima wanneer je zeker weet dat al je form elementen over de hele website er hetzelfde uit komen te zien. Je had dit dan wel bovenin je CSS mogen declareren, omdat dit dan je generieke styling is.